### PR TITLE
Add device count, selected count and select all checkbox

### PR DESCRIPTION
**Pull request by lawik**

### CODE REVIEW

**Comment by lawik** [comment]
I just do quality of life improvements now. This is my life.

File: lib/nerves_hub_web/live/devices/index.ex [hunk 1]
```diff
@@ -199,6 +199,21 @@ defmodule NervesHubWeb.Live.Devices.Index do
     {:noreply, assign(socket, :selected_devices, selected_devices)}
   end
 
+  def handle_event("select-all", _, socket) do
+    selected_devices = socket.assigns.selected_devices
+
+    dbg(selected_devices)
```
Comment (joshk) [inline lib/nerves_hub_web/live/devices/index.ex:205]: you might want to remove this

**Review by joshk — APPROVED** [review]

File: lib/nerves_hub_web/live/devices/index.ex (hunk 1 above)
Comment (lawik) [reply lib/nerves_hub_web/live/devices/index.ex]: yes :D

**Comment by joshk** [comment]
Is it worth having a test for this?

**Pull request merged**